app/src/components/ReviewElement.tsx

```diff
@@ -1,9 +1,11 @@
 import { Trans } from 'next-i18next'
 import { ReactElement } from 'react'
 
+import { i18nKey } from '@src/types'
+
 export type ReviewElementProps = {
-  labelKey: string
-  children: ReactElement | null
+  labelKey: i18nKey
+  children: ReactElement | string | null
```

Comment:
Notes for reviewers: Turned out I needed to sometimes have a `<ReviewElement>` whose child is just a string and not a full ReactElement.